### ISSUE TITLE
Training issue on Vimeo Dataset

### ISSUE THREAD

**Issue by daisily727** [myungsub/CAIN#11]
Hi, thanks for sharing the code. I tried to train on Vimeo dataset from scratch just use your script but seems it stucks at PSNR around 15 and SSIM around 0.5. Is there any special recipe that I am mising in the training process?

**Comment by myungsub** [comment]
Hi, thanks for your interest in our work.
There shouldn't be any special recipes.. could you tell us about your problem more in detail?
Does our pretrained model work fine in your running environment?

**Comment by HubertSotnowski** [comment]
Hi, Training cain on new PyTorch don't work correctly. Try PyTorch 1.5.

**Comment by daisily727** [comment]
> Hi, Training cain on new PyTorch don't work correctly. Try PyTorch 1.5.

Thanks for the reply. I have met the same. I got it work correctly now by changing from 1.7 to 1.4

**Comment by Kirstihly** [comment]
Add `torch.manual_seed(seed)` to data augmentation may be one of the fix, as mentioned in [Bugfix/random-seed-for-pytorch>=1.6](https://github.com/myungsub/CAIN/pull/25)
